docs/installation/install-ksql-with-docker.rst

```diff
@@ -182,6 +182,7 @@ Run a KSQL Server that enables manual interaction by using the KSQL CLI:
 
 ``KSQL_LISTENERS``
     A list of URIs, including the protocol, that the broker listens on.
+    If you are using IP6 then you should set to ``http://[::]:8088``.

```

Comment:
```suggestion
    If you are using IP6, use ``http://[::]:8088``.
```

Reply:
nit: can we add a noun/pronoun? `If you are using IPv6, set *this* to...` Similarly for the others below.